src/ontology/templates/assays.tsv

```diff
@@ -307,6 +307,8 @@ OBI:0002441	physical examination of individual			"An assay that produces a descr
 OBI:0002442	hydroxyl-radical footprinting assay			"A footprinting assay that uses the reaction of hydroxyl radicals with side-chain sites in molecules (proteins, DNA, etc)_ with the resultant mass shift demonstrating the site of modification, used to_ to asses the accessibility of that site."	PMID:21770468			Jie Zheng			subclass		footprinting assay			molecular entity			oxygen radical						
 OBI:0002443	methidiumpropyl-EDTA-iron(II) footprinting assay		MPE-Fe(II) footprinting	A footprinting assay that determines protein-binding sites on DNA by partial cleavage of ligand-protected DNA restriction fragments with methidium-propyl-EDTA (MPE). MPE-Fe(II) in the presence of oxygen efficiently catalyzes the non-specific clevage of DNA.	PMID:6225070|PMID:6435669			Jie Zheng			subclass		footprinting assay			DNA extract			EDTA methidiumpropylamide			binding datum			
 OBI:0002445	transcription start site mapping by primer extension assay			"A transcription profiling assay in which the transcription start site for a gene is determined by identifying the 5' end of mRNA. A radio-labeled primer is annealed to a complementary mRNA sequence near the 3' end, then cDNA is synthesized until the 5' end is reached."	PMID:23378648|url:https://en.wikipedia.org/wiki/Primer_extension			Rebecca Jackson			subclass		transcription profiling assay								RNA extract	sequence data	gene expression	transcription start site identification objective	
+OBI:0002447	suppression subtractive hybridization		SSH|subtractive hybridization	"An artificially induced nucleic acid hybridization that is performed to compare gene expression in different cell or tissue types based on normalization and suppression, which creates a subtracted cDNA or genomic DNA library."	PMID:14970460			Rebecca Jackson			subclass		artificially induced nucleic acid hybridization												
+OBI:0002448	differential screening hybridization		DSH|differential screening	An artificially induced nucleic acid hybridization that is performed to identify differentially expressed genes through hybridization of cDNA probes.	PMID:25472628			Rebecca Jackson			subclass		artificially induced nucleic acid hybridization												
```

Comment:
These two are not assays? I see them under 'material processing'. There should be a check to catch this.